numpy/core/records.py

```diff
@@ -606,6 +628,17 @@ def fromarrays(arrayList, dtype=None, shape=None, formats=None,
     >>> x1[1]=34
     >>> r.a
     array([1, 2, 3, 4])
+    
+    >>> x1 = np.array([1,2,3,4])
+    >>> x2 = np.array(['a','dd','xyz','12'])
+    >>> x3 = np.array([1.1,2,3,4])
+    >>> r = np.core.records.fromarrays(
+            [x1, x2, x3], names='d,e,f',
```

Comment:
```suggestion
...         [x1, x2, x3], names='d,e,f',
```
And on the line below